app/helpers/routing_helper.rb

```diff
@@ -6,6 +6,18 @@ def self.format_seconds_as_minsec(sec)
     format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
   end
 
+  def self.coordinates(input)
+    return input if valid_coordinates?(input)
+
+    Places::DESTINATIONS.dig(input, :location)
+  end
+
+  def self.center(start_coordinates)
+    return [52.39339, 13.13208] if start_coordinates.nil?
+
+    start_coordinates.split(",").map(&:to_f)
```

Comment:
Any reason why we are converting to float here? In the earlier version, we used `@target.split(",")` which would be strings.

32bit floats have about [7 decimal digits](https://stackoverflow.com/a/13543600/13679671) of precision, I don't know whether a) Ruby "float" is actually a C "double" and b) how much precision we need. If that would become a problem, I would prefer it if we stick to strings for now, but I am not sure if it will :)

Reply:
You're right. `leaflet-rails` doesn't really care about whether it receives a float or a string here, so there's no real reason for the conversion. I changed this (as well as the guard clause) to return strings again.